services/security/src/app/features/policy/handlers/add-policy.handler.ts

```diff
@@ -17,6 +19,14 @@ export default class AddPolicyHandler implements Handler<AddPolicyCommand> {
     const { attribute, resource } = command.payload;
 
     const newPolicy = await policyRepository.addPolicy(PolicyModel.create({ attribute, resource }));
+    await this.dependencies.eventDispatcher.dispatch({
+      name: "PolicyAdded",
```

Comment:
It would be better to have events as a classes, fully typed

Reply:
Done